lib/code_ownership/private.rb

```diff
@@ -81,6 +81,19 @@ def self.tracked_files
       @tracked_files ||= Dir.glob(configuration.owned_globs) - Dir.glob(configuration.unowned_globs)
     end
 
+    sig { params(file: String).returns(T::Boolean) }
+    def self.file_tracked?(file)
+      in_owned_globs = configuration.owned_globs.all? do |owned_glob|
+        File.fnmatch?(owned_glob, file, File::FNM_PATHNAME | File::FNM_EXTGLOB)
+      end
+
+      in_unowned_globs = configuration.unowned_globs.all? do |unowned_glob|
+        File.fnmatch?(unowned_glob, file, File::FNM_PATHNAME | File::FNM_EXTGLOB)
+      end
```

Comment:
Might be worth adding a comment about the fnmatch vs globbing.

Reply:
Good call, added!